README.md

```diff
@@ -137,7 +137,7 @@ Fencer-specific environment variables are:
 - `RUNTIME_IGNOREDOTFILES` - A flag indicating whether to ignore files
   with names starting with a dot (hidden files on Linux-based systems
   and macOS). It can be `True` or `False`. The default value is
-  `False`.
+  `False`. Directories with names starting in a dot are not ignored.
```

Comment:
They are.

So, this line should be removed and the description should say "A flag indicating whether to ignore files and directories"

Reply:
Fixed. Thank you for spotting this!